config/owasp/suppressions.xml

```diff
@@ -41,8 +41,6 @@
     <cve>CVE-2022-3064</cve>
     <cve>CVE-2022-45143</cve>
     <cve>CVE-2023-20883</cve>
-    <cve>CVE-2023-34034</cve>
-    <cve>CVE-2023-39017</cve>
```

Comment:
lets keep it in sync with master unless confident. PET team is the owner of this file.

Reply:
made suppression in synch with master